frontend/src/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
is this what renders the frontend now?

Reply:
import React from 'react'; is from react app